python/snewpy/snowglobes_interface.py

```diff
@@ -239,11 +261,14 @@ def run(self, flux_files, detector:str, material:str=None):
             raise ValueError(f'Material "{material}" is not in {self.materials}')
         if isinstance(flux_files,str):
             flux_files = [flux_files]
-        
+
         with tqdm(total=len(flux_files), leave=False, desc='Flux files') as progressbar:
             results = []
             for flux_file in flux_files:
-                result = self._compute_rates(detector,material,Path(flux_file))
+                e_flux = np.loadtxt(Path(flux_file).resolve()).T
+                energies = e_flux[0]
+                fluxes = e_flux[1:]
+                result = self.compute_rates(detector, material, fluxes, energies)
```

Comment:
I also agree with this change as discussed in the issue, better fluxes and energies than the path, and removing the underscore